legend-engine-language-pure-dsl-service/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/ServiceParserGrammar.g4

```diff
@@ -230,4 +231,28 @@ testSingleValueParam:                        primitiveValue
 ;
 
 testParam:                              testListValueParam | testSingleValueParam
+;
+
+// ----------------------------------- EXECUTION_ENVIRONMENT ------------------------------------------------------
+execEnvs:                               EXEC_ENV qualifiedName
+                                            BRACE_OPEN
+                                                execEnvType
+                                                executions
+                                            BRACE_CLOSE
+;
+execEnvType:                            EXEC_ENV_TYPE COLON execEnvTypes SEMI_COLON
+;
+execEnvTypes:                           SERVICE_SINGLE| SERVICE_MULTI
```

Comment:
This is kind of confusing - Multi exec service needs SingleExecEnv. May be we need a different name?

Reply:
well lexer doesnt alloy adding two different names with the same value

Reply:
renaming it SINGLE_KEYWORD and MULTI_KEYWORD